### DISCUSSION TITLE
Page renders at 500% zoom

### DISCUSSION THREAD

**Discussion by kbmetz** [investbrainapp/investbrain#7] [Q&A] [answered]
I've used docker compose up and everything is running. Made an account and I can successfully log in. I've tried from 2 different browsers and my phone and the site renders like it's zoomed in 500%. It's not usable at all. Not sure if I missed some step?

**Comment by hackeresq** [comment]
Hi @kbmetz are you still having this issue?  

I'm wondering if something is off with NPM building the front-end resources. Hmm.

Can you share a screenshot? Also, what OS and processor architecture are you running for your Docker host machine?

**Reply by kbmetz** [reply]
Also just for an FYI, I'm running this on a Ubuntu 22.04 server with docker version 27.3.1 on 
Linux garage-01 6.8.0-48-generic #48~22.04.1-Ubuntu SMP PREEMPT_DYNAMIC Mon Oct  7 11:24:13 UTC 2 x86_64 x86_64 x86_64 GNU/Linux

**Comment by Phaeton** [comment, marked as answer]
It was a CORS issue for me. Updating the `APP_URL`  in the `.env` file from `"http://localhost:${APP_PORT}"` to `http://192.0.2.1:${APP_PORT}"` fixed the scaling issue. Make sure the IP address matches your server address.

**Reply by hackeresq** [reply]
That likely it. I'll have to check if the CORS configuration is too restrictive. In any case, I'll update the docs to reference this as a troubleshooting step! Thanks for jumping in to help, @Phaeton!

**Reply by hackeresq** [reply]
This is resolved with the change in #10!

**Reply by kbmetz** [reply]
Yes! This fixes the issue for me. Thanks!
